Make.ps1

```diff
@@ -0,0 +1,163 @@
+<#
+.SYNOPSIS
+    Run on Windows the same commands as in ./Makefile without installing any aditional software.
```

Comment:
Overall looks good, we may need to update README or development guide on how to use the new Make.ps1 and at the same time look at other mentions to ensure community knows to keep both the original Makefile and Make.ps1 in sync.

Reply:
Good point! I'll update those too.

Reply:
Done